### PR TITLE
More proofreading (because why not)

### PR DESCRIPTION
**Pull request by PlutarianReal**
Title explains it all

### CODE REVIEW

**Comment by PlutarianReal** [comment]
I think I might need an apostrophe somewhere, but I do not know.

File: help/articles/101013/index.html [hunk 1]
```diff
@@ -37,8 +37,8 @@ <h3>Why reporting content is important</h3>
...
         <div class="info">
-            <p>We have a Zero Tolerance policy for any kinds of 18+/sexually explicit (adult) contents.</p>
-            <p>We do not allow for inappropriate Profile Pictures, Posts (Images or Text Descriptions), Usernames, Banners, User Biographies, external links or sexually explicit Private Messages, including 18+ Adult Contents, Sexually Explicit Contents (including Images in Posts, Private Messages and Banners, Text Descriptions of Posts or Private Messages) or Illegal Substances (images, references and discussions).
+            <p>We have a Zero Tolerance policy for any kinds of 18+/sexually explicit (adult) content.</p>
+            <p>We do not allow inappropriate Profile Pictures, Posts (Images or Text Descriptions), Usernames, Banners, User Biographies, external links or sexually explicit Private Messages, including 18+ Adult Contents, Sexually Explicit Contents (including Images in Posts, Private Messages and Banners, Text Descriptions of Posts or Private Messages) or Illegal Substances (images, references and discussions). In addition, do not infringe on others copyright.
```
Comment (SleepingAmi) [inline help/articles/101013/index.html:41]: I believe copyright and DMCA issues should be in another article, and therefore, I would like to request that you revert the changes related to it.

File: help/articles/101013/index.html [hunk 2]
```diff
@@ -121,15 +121,15 @@ <h3>Reporting Private Messages (PM's)</h3>
...
+            <li>Click on the "Report Message" option from the Dropdown Menu.</li>
             <li>Follow along the walkthrough, <b>making sure you read it <span style="text-decoration: underline;">carefully</span></b>.</li>
             <li>You are all set. Please note that as stated in the walkthrough, the content of the message you reported will be sent, <span style="text-decoration: underline;">unencrypted</span>, to our Review Team, along with the ID of the message, message chat and the user who <span style="text-decoration: underline;">sent</span> the message.</li>
         </ol>
         <br>
         <h3>On the Mobile Clients:</h3>
         <ol>
             <li>Long press on the message you wish to report.</li>
-            <li>Press on the "Report Message" option from the Dropdown Menu.</li>
+            <li>Click on the "Report Message" option from the Dropdown Menu.</li>
```
Comment (SleepingAmi) [inline help/articles/101013/index.html:132]: As @3ux4 mentioned, you can't (normally) click on a button on a mobile client. That's why we worded it as "press," which is understood as clicking where touching is not an option.

Feel free to leave the "click" correction on PC (bear in mind touchscreen laptops and people who use touchpads), but the mobile client's "click" is redundant and (under normal circumstances, considering most people don't use their phone/tablet as a PC or don't have damaged touchscreens which require them to use a mouse or other external entry device)

**Review by SleepingAmi — CHANGES_REQUESTED** [review]

**Review by SleepingAmi — APPROVED** [review]
I also approve this bc I have to

**Comment by SleepingAmi** [comment]
happy coding!
